src/services/item/plugins/app/service.ts

```diff
@@ -94,7 +99,12 @@ export class AppService {
   }
 
   // used by app : get tree
-  async getItemsByType(actorId: UUID, repositories: Repositories, types: ItemType[], path: string) {
+  async getItemsByType(
+    actorId: UUID,
+    repositories: Repositories,
+    types: UnionOfConst<typeof ItemType>[],
```

Comment:
Info: what does this type change do?

Reply:
Previously `ItemType` was an `enum`. I changed it to be a const object which means that to obtain the union of literal values before we did 
```ts
`${ItemType}
```
but now we need to do:
```ts
(typeof ItemType)[keyof typeof ItemType]
```
`UnionOfConst<T>` is just a generic utility type I defined in sdk to help with this conversion.